docs/specs/oidc.md

```diff
@@ -461,44 +470,114 @@ Silent Authentication
 1. If authorization code request result is successful, send the token request to Authgear with code + code verifier
 1. Authgear returns token response to SDK with id token + new access token
 
-## First-party Clients
+## Clients
+
+The value of `x_application_type` determines the type of the client.
 
-OAuth protocol allows user to delegate access to clients, so that clients can
-act on behalf of the user.
+The tables summarizes client Party, confidentiality, PII in ID token, and access to privileged user operations of different `x_application_type`.
 
-Usually, OAuth clients are third-party clients, with limited trust:
-- User need to give consent to a client before clients is allowed access.
-- Privileged user operations (e.g. change password) is usually not exposed
-  through APIs accessible through OAuth protocol.
+|`x_application_type`|Client Party|Confidentiality|PII in ID token|Access to privileged user operations|
```

Comment:
Can we add reason behind the difference of 'PII in ID token' and 'Access to privileged user operations' between different kinds of client?

Reply:
Actually they are inverse of each other. I listed them separately in the table with the intention to make things clearer. Maybe I should add a paragraph to explain their logical relationship?

Reply:
I think we would need to explain the rationales of the limitations:
- why confidential clients cannot use re-authentication?
- why only public clients have access to privileged user operations?

Reply:
Added a section to explain.